CMakeLists.txt

```diff
@@ -584,7 +584,7 @@ if(LIBUV_BUILD_TESTS)
            WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
 endif()
 
-if(UNIX OR MINGW)
+if(UNIX OR MINGW OR MSVC)
```

Comment:
It appears to me this conditional should now just be removed, as this is all platforms, but otherwise SGTM

Reply:
@vtjnash I updated the PR to remove the conditional as you suggested.